sql/mssql/inspect.go

```diff
@@ -367,6 +391,17 @@ ORDER BY
 )
 
 type (
+	// MemoryOptimized attribute describes if the table is memory-optimized or disk-based.
+	MemoryOptimized struct {
+		schema.Attr
+		V bool
```

Comment:
```suggestion
```

I think you can remove `V`. Attaching this attribute to a table indicates that it's memory-optimized.